### PR TITLE
ID:1w3hg0t5tybalkp Deploy Yearn.finance on Akash

### PR DESCRIPTION
**Pull request by halw001**
{
"services": {
"web": {
"name": "web",
"available": 1,
"total": 1,
"uris": [
"8s72s6tabpce3acsbd4ihe6g5o.provider4.akashdev.net"
],
"observed-generation": 0,
"replicas": 0,
"updated-replicas": 0,
"ready-replicas": 0,
"available-replicas": 0
}
},
"forwarded-ports": {}
}

### CODE REVIEW

**Comment by halw001** [comment]
pull a new request

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Awesome DeFi apps you can deploy on Akash
 - [Balancer](Balancer)
 - [Luaswap](luaswap)
 - [SushiSwap](SushiSwap)
+- [Yearn.finance](https://github.com/iearn-finance/iearn-finance)
```
Comment (boz) [inline README.md:30]: Please link to the directory here like the others do.  Thanks!

**Review by boz — CHANGES_REQUESTED** [review]
Looks great!  Change the link in the readme and we'll merge.  Thanks!

File: README.md (hunk 1 above)
Comment (halw001) [reply README.md]: OK,i have changed the link.

Comment (boz) [reply README.md]: it's still not quite right.  Look at sushi swap above.  it should link to `Yearn.finance` or whatever the name of the directory you created here is.

Comment (halw001) [reply README.md]: OK,i have changed the link.

**Pull request merged**